app/containers/MessageActions.js

```diff
@@ -311,6 +317,16 @@ export default class MessageActions extends React.Component {
 		replyInit(actionMessage, true);
 	}
 
+	handleUnread = () => {
+		const { actionMessage } = this.props;
+		try {
+			RocketChat.toggleRead(true, undefined, actionMessage._id);
+			Navigation.navigate('RoomsListView');
```

Comment:
Not sure that navigation to RoomsListView should occur after message is marked as read\unread. 
@diegolmello what do you think?

Reply:
It's the same behaviour of web version.
I think it's ok.